webapp/src/components/training/Trainer.vue

```diff
@@ -24,6 +27,9 @@
               @click="startTraining(true)"
             >
               train collaboratively
+              <template #description>
+                Share the model's weights with other participants
```

Comment:
```suggestion
                Exchange model updates with other participants
```
I would use less technical terms for non-technical users